src/main/java/models/Game.java

```diff
@@ -0,0 +1,72 @@
+package models;
+
+import java.lang.reflect.Array;
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class Game {
+
+    private static final Game GAME_INSTANCE = new Game();
+
+    private ArrayList<Player> allPlayers;
+    private ArrayList<Team> allTeams;
+    private League league;
```

Comment:
**[IMPORTANT]** Making this an array and allowing for more than one league would help your code accommodate changes and addition easier later. This is one of the core evaluation points of this project (See TeamProjectGuide)